### PR TITLE
add function for ui button

### PR DESCRIPTION
**Pull request by liamcindy**

### CODE REVIEW

**Comment by liamcindy** [comment]
@minggo please reivew this pr,thanks

File: cocos/ui/UIButton.h [hunk 1]
```diff
@@ -212,6 +212,7 @@ class CC_GUI_DLL Button : public Widget
                       const std::string& disableImage = "",
                       TextureResType texType = TextureResType::LOCAL);
 
+    virtual Size getNormalTextureSize() const;
```
Comment (zilongshanren) [inline cocos/ui/UIButton.h:215]: Does this function only available for internal usage?

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by liamcindy** [comment]
@andyque This function should be public

**Comment by zilongshanren** [comment]
If so , @liamcindy   could you change it to public?
Thanks.

**Comment by zilongshanren** [comment]
@minggo 
This PR is ready to be merged.

**Pull request merged**